test/jasmine/tests/lib_test.js

```diff
@@ -1984,6 +1984,109 @@ describe('Test lib.js:', function() {
             });
         });
     });
+
+    describe('relinkPrivateKeys', function() {
+        it('ignores customdata and ids', function() {
+            var fromContainer = {
+                customdata: [{_x: 1, _y: 2, a: 3}],
+                ids: [{_i: 4, j: 5}]
+            };
+            var toContainer = {
+                customdata: [{a: 6}],
+                ids: [{j: 7}]
+            };
+
+            Lib.relinkPrivateKeys(toContainer, fromContainer);
+
+            expect(toContainer.customdata[0]._x).toBeUndefined();
+            expect(toContainer.customdata[0]._y).toBeUndefined();
+            expect(toContainer.ids[0]._i).toBeUndefined();
+        });
+
+        it('ignores any values that are ===', function() {
+            var accesses = 0;
+
+            var obj = {
+                get _x() { accesses++; return 1; },
```

Comment:
Really, this is valid ES5?

Reply:
Yes, I believe so, though don't use it in `src` as it's not supported by IE or Safari. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

Reply:
Oh actually I was reading the browser compatibility table wrong... in fact looks like this basic setter/getter usage is fine even in IE and Safari, though I still feel like we should avoid it as it can  make things confusing. I've only ever used it for testing and debugging. If, for example, a property gets set but you can't figure out *where* it's getting set, attach a setter for that property that throws an error.